data/tlg0018/tlg002/__cts__.xml

```diff
@@ -1,8 +1,8 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0018" xml:lang="grc" urn="urn:cts:greekLit:tlg0018.tlg002">
   <ti:title xml:lang="lat">Legum Allegoriarum Libri I-III</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0018.tlg002.opp-grc1" workUrn="urn:cts:greekLit:tlg0018.tlg002" xml:lang="grc">
+  <ti:edition urn="urn:cts:greekLit:tlg0018.tlg002.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0018.tlg002" xml:lang="grc">
     <ti:label xml:lang="lat">Legum Allegoriarum Libri I-III</ti:label>
-    <ti:description xml:lang="mul">Philo Judaeus. Cohn, Leonard, editor. Opera quae supersunt, Volume 1. Berlin: Reimer, 1896.</ti:description>
+    <ti:description xml:lang="mul">Philo Judaeus. Opera quae supersunt, Cohn, Leonard, editor. Volume 1. Berlin: Reimer, 1896.</ti:description>
```

Comment:
Looks like the Volume is in the wrong place?

Reply:
Thought I had caught them al! Thanks!